bentoml/_internal/bento/bento.py

```diff
@@ -131,7 +137,10 @@ def create(
         # Create `readme.md` file
         with bento_fs.open("README.md", "w") as f:
             description = description if description else svc.__doc__
-            f.write(description)
+            if description is None:
+                logger.warning(f"no description found while creating Bento {tag}")
```

Comment:
there's no need to warn user about this, we can just do a `if description is not None` check here